### PR TITLE
add .net sdk preview3 fsprojs to core3

### PR DESCRIPTION
**Pull request by enricosada**
using .net core sdk `1.0.0-preview3-004007`

To test:

- build paket `build` (becuase some paket files are rrequired as sources)
- `cd src\Paket.Core`
- `dotnet restore Paket.Core.preview3.fsproj`
- `dotnet build Paket.Core.preview3.fsproj`

### CODE REVIEW

**Comment by enricosada** [comment]
Atm i get one error

```
         ProjectFile.fs(1341,35): error FS0039: The record label or namespace 'PackagesConfigFile' is not defined [e:\github\Paket\src\Paket.Core\Paket.Core.preview3.fsproj]
```

same error if i build `src\Paket.Core\Paket.Core` project.json with `preview2`, so is just something new?

**Comment by enricosada** [comment]
Nvm fixed :D

- `Paket.Core.preview3` is ok

works with `dotnet pack Paket.Core.preview3.fsproj` too

**Comment by forki** [comment]
rebased it on master and will take a look tomorrow. thanks buddy

File: src/Paket/Program.fs [hunk 1]
```diff
@@ -207,7 +211,11 @@ let pack (results : ParseResults<_>) =
                       ?releaseNotes = results.TryGetResult <@ PackArgs.ReleaseNotes @>,
                       ?templateFile = results.TryGetResult <@ PackArgs.TemplateFile @>,
                       excludedTemplates = results.GetResults <@ PackArgs.ExcludedTemplate @>,
+#if NETCOREAPP1_0
+                      workingDir = System.IO.Directory.GetCurrentDirectory(),
```
Comment (forki) [inline src/Paket/Program.fs]: jupp will do

**Comment by forki** [comment]
ok this is pretty cool.
Restore and Build seem to work on master. @enricosada could you please take a look on Pack? I commented it out for now

image

**Comment by forki** [comment]
Microsoft.FSharp.Core.netcore is that stuff merged into FSharp.Core alpha package?
/cc @dsyme @enricosada @KevinRansom

**Comment by enricosada** [comment]
Try workaround for pack https://github.com/dotnet/netcorecli-fsc/blob/master/examples/preview3/console-crossgen/README.md

Yes is the same. Use fsharp.core if you want.

**Comment by enricosada** [comment]
There are issues with fsharp.core a netcore but less discuss later.

**Comment by forki** [comment]
image

**Comment by enricosada** [comment]
@forki i'll check later. 
Workaround was for crossgen (net46+netcoreapp). 
I need to add the use case `dotnet pack` of console netcoreapp to netcorecli-fsc test suite too :)

**Comment by forki** [comment]
Cool. I'll commented it out for now.

Am 05.11.2016 12:58 nachm. schrieb "Enrico Sada" <notifications@github.com>:

**Comment by forki** [comment]
https://ci.appveyor.com/project/SteffenForkmann/paket/branch/master 

    The specified deps.json [C:\Users\appveyor\.nuget\packages\.tools\dotnet-compile-fsc\1.0.0-preview2-020000\netcoreapp1.0\dotnet-compile-fsc.deps.json] does not exist

what does that mean?

**Comment by forki** [comment]
<PackageReference Include="FSharp.NET.Sdk">
      <Version>1.0.0-alpha-000001</Version>
      <PrivateAssets>All</PrivateAssets>
    </PackageReference>

what does PrivateAssets mean and is it needed?

**Comment by enricosada** [comment]
It's new of nuget spec for references AFAIK.  
check the generated nuspec inside package. 
I see Three values: Code(something like that)/analyzer/build. 
All mean to restore also build (msbuild targets i think). 
I didnt searched more details becuase doesnt matter for me but i think works like that. 
If you Explore more and find more info, pls send a link to me too.

**Comment by enricosada** [comment]
Deps.json error dunno why happen. I'll check

**Pull request merged**